dev/openchpl_views.sql

```diff
@@ -560,8 +561,21 @@ LEFT JOIN
 LEFT JOIN
   (SELECT vendor_1.vendor_id,
           vendor_1.name AS vendor_name,
-          vendor_1.vendor_code
+          vendor_1.vendor_code,
+          vendor_1.vendor_status_id
    FROM openchpl.vendor vendor_1) vendor ON product.vendor_id = vendor.vendor_id
+     LEFT JOIN ( SELECT vshistory.vendor_status_id,
+            vshistory.vendor_id,
+            vshistory.status_date AS last_vendor_status_change
+           FROM openchpl.vendor_status_history vshistory
+             JOIN ( SELECT vendor_status_history.vendor_id,
+                    max(vendor_status_history.status_date) AS status_date
+                   FROM openchpl.vendor_status_history
+                  WHERE vendor_status_history.deleted = false
+                  GROUP BY vendor_status_history.vendor_id) vsinner ON vshistory.vendor_id = vsinner.vendor_id AND vshistory.status_date = vsinner.status_date) vendorstatus ON vendorstatus.vendor_id = vendor.vendor_id
+     LEFT JOIN ( SELECT vendor_status.vendor_status_id,
+            vendor_status.name AS vendor_status_name
+           FROM openchpl.vendor_status) v ON vendorstatus.vendor_status_id = v.vendor_status_id
```

Comment:
Can you pick a more descriptive alias than 'v'? The other joined tables have something a little bit more verbose and I think it helps with these complicated queries.